src/main/java/website/frontrow/Launcher.java

```diff
@@ -62,7 +62,7 @@ public static void main(String[] args) throws IOException
      * @param filename The file name of the level to load.
      * @param playerCount The amount of players in this game.
      */
-    @SuppressWarnings("methodlength") // We need to make a GameFactory and UIBuilder
+    @SuppressWarnings("methodlength")
```

Comment:
If there's a factory now, does this methodlength still trigger in checkstyle? Because if it does the method will have to be made shorter, otherwise remove this Suppression.

<layout